.github/workflows/php-unit-tests.yaml

```
@@ -318,7 +318,7 @@
       - name: Set up PHP
         uses: shivammathur/setup-php@v2
         with:
-          php_version: "8.3"
+          php_version: "8.4"
```

actionlint (1.7.7)

input "php_version" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)